components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/APISubscriptionDAOImpl.java

```diff
@@ -136,8 +137,8 @@ public Subscription getAPISubscription(String subscriptionId) throws APIMgtDAOEx
                 return createSubscriptionsWithApiInformationOnly(rs);
             }
         } catch (SQLException e) {
-            log.error("Error while executing sql query", e);
-            throw new APIMgtDAOException(e);
+            throw new APIMgtDAOException(DAOUtil.DAO_ERROR_PREFIX +
+                    "getting api subscriptions by application(appId: " + applicationId + ")", e);
```

Comment:
Should we pass APP id in error response? What if we pass application name as well here?

Reply:
This wont go in the error response, this string will simply get printed in the logs when exception is logged